components/FlexibleAnswers/FlexibleAnswers.tsx

```diff
@@ -0,0 +1,81 @@
+interface RepeaterGroupAnswer {
+  [key: string]: string | string[];
+}
+
+const RepeaterGroupAnswer = ({
+  data,
+}: {
+  data: RepeaterGroupAnswer;
+}): React.ReactElement => (
+  <ul className="govuk-list lbh-list">
+    {Object.keys(data).map((questionName) => (
+      <li key={questionName}>
+        <strong>{questionName}:</strong>{' '}
+        {Array.isArray(data[questionName])
+          ? (data[questionName] as string[]).join(', ')
+          : data[questionName]}
+      </li>
+    ))}
+  </ul>
+);
+
+const RepeaterGroupAnswers = ({
+  data,
+}: {
+  data: (string | RepeaterGroupAnswer)[];
+}): React.ReactElement => (
+  <ul className="govuk-list lbh-list">
+    {data.map((item, i) => (
```

Comment:
If this is a repeater group can new items be added, can items be removed?

If so we need to be careful with using the index as a key.

However do we have anything appropriate here that was could use as a unique identifier for each element.